app/utils/CheckYourAnswersHelper.scala

```diff
@@ -107,7 +104,4 @@ class CheckYourAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messag
     }
 }
 
-object CheckYourAnswersHelper {
-
-  private val dateFormatter = DateTimeFormatter.ofPattern("d MMMM yyyy")
-}
+object CheckYourAnswersHelper
```

Comment:
I suppose that formatter is not used if deleted?

Reply:
**We are using this `dateFormatter` in two places, but not sure whether we use them or not:**
```
https://github.com/hmrc/amls-estate-agency-business-frontend/blob/master/.g8/datePage/migrations/%24className__snake%24.sh#L60
https://github.com/hmrc/amls-estate-agency-business-frontend/blob/master/migrations/applied_migrations/DateOfChange.sh#L60
```

Reply:
Maybe is good to keep it for safety?